src/index.html

```diff
@@ -3,144 +3,144 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- fonts -->
     <link
-      href="//db.onlinewebfonts.com/c/c862d8098ce50a7dc042ebbb9b94bbc8?family=TwCenW01-Medium"
       rel="stylesheet"
+      media="screen"
+      href="https://fontlibrary.org/face/germano"
       type="text/css"
     />
-    <link rel="stylesheet" href="./css/style.scss" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=IBM+Plex+Serif:ital,wght@0,100;1,100&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=IBM+Plex+Serif:ital,wght@0,100;1,100&display=swap"
+      rel="stylesheet"
+    />
+    <!-- icons -->
     <link
       rel="stylesheet"
       media="screen"
-      href="https://fontlibrary.org/face/germano"
+      href="./icons/font/flaticon.css"
       type="text/css"
     />
-    <link rel="stylesheet" href="./icons/font/flaticon.css" />
     <link
       rel="stylesheet"
       href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/all.min.css"
       integrity="sha512-+4zCK9k+qNFUR5X+cKL9EIR+ZOhtIloNl9GIKS57V1MyNsYpYcUrUeQc9vNfzsWfV28IaLL3i96P9sdNyeRssA=="
       crossorigin="anonymous"
     />
+    <link rel="stylesheet" href="./css/style.scss" />
     <script
       type="text/javascript"
       src="https://npmcdn.com/parse/dist/parse.min.js"
     ></script>
-    <!-- <link rel="stylesheet" href="./css/popups.css" /> -->
+    <script src="https://unpkg.com/magic-grid/dist/magic-grid.cjs.js"></script>
```

Comment:
I see this import, but didn't find the code to implement the actual magicGrid. did you push it?

Reply:
No yet, I'm doing it today

Reply:
I pushed it, as you'll see it didn't work. Not stating a number of items throws a missing property error

Reply:
It seems to work. I resize the page and the elements change position. And you also passed the missing items length. I see though, that removing an item breaks the layout. Why exactly do you wanted to use this plugin? It has a major bug related with [dynamic images height](https://github.com/e-oj/Magic-Grid/issues/47)

Reply:
Oh, maybe you'll have images with very different heights, right? Gotcha... What you are looking for is called [Masonry layout](https://www.google.com/search?q=masonry+layout+css). Maybe you could try another package?

Reply:
Exactly. I already tried others and again this morning but turns out there are not that many dynamic content masonary libraries. I tried gridstack, macy.js and bricklayer. Somewhat better results compared to magic grid (no height issue) but worse than pure flex and none work well when adding/removing items.

Reply:
I don't know any other method for this type of layout 🤔 Have you tried https://masonry.desandro.com? it seems the most popular.

Reply:
I think it needs static elements but I can't remember where I got that from, does it look like that to you? Did you maybe try this out with MagicGrid on your computer? There's positionElements() that I tried to use but it didn't work.